### ISSUE TITLE
[Raspberry4] ERROR: Failed to initialise renderer

### ISSUE THREAD

**Issue by 3kynox** [any1/wayvnc#44]
Hello folks,

got an error after compiled wayvnc (and deps) on a raspberry4. Here is the details of my config:

Device: Raspberry4B 4GB
System: Archlinux on ArmV8
WM: Sway

Once the build is complete, this error is reported:
```
DEBUG: OpenGL ES 3.1 Mesa 20.0.7
ERROR: Failed to initialise renderer
```

@any1 started to guide me throught a possible resolution with no luck at this time.

Thanks anyway about your thoughts

**Comment by waligu** [comment]
It is the same error i have - i compiled today the new versions cause i have seen there are some new improvements but it is still not working. How can we move forward to fix this error?

**Comment by waligu** [comment]
i use the new intel-media-driver - called iris or short iHD 20.2.pre and libva 2.8.0.pre1

**Comment by any1** [comment]
I have added some more detailed error messages since this was reported. Does it not say anything more than "Failed to initialise renderer"?

**Comment by waligu** [comment]
No same message - nothing more on the screen.

**Comment by waligu** [comment]
ERROR
DEBUG: OpenGL ES 3.2 Mesa 20.2.0-devel (git-8c962f5f61)
ERROR: Failed to initialise renderer

in kodi mesa works and plays videos

**Comment by any1** [comment]
And you've definitely pulled the newest commits and compiled from the master branch?

**Comment by any1** [comment]
You could try this branch where the renderer has been removed completely: https://github.com/any1/wayvnc/tree/pixman-rendering

It's work in progress but it should do the job.

**Comment by waligu** [comment]
i try

**Comment by waligu** [comment]
no error message anymore and starting.
but after connecting a neatvnc error in memory drops the connection
taking a look there - maybe my viewer is not the correct one.

**Comment by any1** [comment]
Should no longer be an issue with the new rendering code. Please open a new issue if your connection keeps dropping.
